libs/coin-modules/coin-sui/src/bridge/getFeesForTransaction.ts

```diff
@@ -32,6 +32,12 @@ export default async function getEstimatedFees({
     }), // Remove fees if present since we are fetching fees
   };
 
-  const fees = await estimateFees(account.freshAddress, t);
+  const fees = await estimateFees({
+    recipient: getAbandonSeedAddress(account.currency.id),
```

Comment:
Why not getting the recipient from the transaction?

Reply:
It was done before my changes, changed to `t.recipient`, but it doesn't change estimation, we need any valid address I think